crates/telio-lana/src/event_log_moose.rs

```diff
@@ -98,109 +99,167 @@ pub fn init_context_info() {
 /// # Parameters:
 /// * foreign_context - &str value containing the json context
 /// # Returns:
-/// * MeshnetappContextDevice - Item containing parsed information from the json context
+/// * MeshnetappContext - Item containing user, app and device information parsed from the json context
```

Comment:
nit: this might go quickly out of date if `MeshnetappContext` ever changes. When looking through generated docs `MeshnetappContext` is already hyperlinked to the structure. tldr: I prefer the previous comment :)